x-pack/test/pki_api_integration/fixtures/kibana_ca.crt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Are we still using this? It looks like we're using the `elasticsearch.crt` and `elasticsearch.key` from the `@kbn/dev-utils`

Reply:
Well, for tests we use the following "trust" configuration:

* ES trusts only `ca.crt` from `@kbn/dev-utils`
* Kibana trusts both `ca.crt` from `@kbn/dev-utils` and `kibana_ca.crt`

`kibana_ca.crt` is used to sign `untrusted_client.p12` so that we can test scenario when client certificate is trusted by Kibana, but not trusted by ES PKI realm. I mentioned this relationship in `fixtures/README.md`, but both description and name of this CA is probably confusing :see_no_evil: 

What do you think?

Reply:
This makes sense now, thanks! I completely overlooked https://github.com/elastic/kibana/pull/42606/files#diff-c60026ca4b4d8f727bc439174688ac5aR61